### DISCUSSION TITLE
找個地方宣洩一下

### DISCUSSION THREAD

**Discussion by Lun603** [share121/inter-knot#5335] [其他]
人生還是太難了，專題同夥完全不做事，還會高高在上的比著我鼻子罵，提的意見一點用都沒有，然後重頭到尾都不做事...又要我匯報進度，我就奇怪了，他到底是我同伴還是我上司?我好累，要不是米家遊戲還在為美好而戰，不然我早就不想相信甚麼美好了。
專題我一個人做到能做的地方，報告論文也是我寫，到底?目前唯一放鬆的方法也就聽聽音，玩玩遊戲了，各位有甚麼方法讓我心裡壓力小點嗎?

**Comment by BJ-cmyk** [comment]
不知道该说什么...
愿所有的美好都能得到祝福
![1726553089286](https://github.com/user-attachments/assets/0b24286d-2b94-4454-a627-c60afd42ee77)

**Reply by Lun603** [reply]
願所有美好都能得到祝福!<img src="https://bbs-static.miyoushe.com/static/2024/08/07/a80f1c174a2b68157a4e8736e376227f_6382287154699407361.png" alt="胖胖鼠-干杯" width="75" >

**Comment by BJ-cmyk** [comment]
找几本书深读，让思想深化，你就会发现为那堆烂事而烦恼根本没必要

**Reply by Lun603** [reply]
最近也有試著讀書轉念，真的挺有用的，但破事還是很多就是了...
